Library/Homebrew/service.rb

```diff
@@ -45,6 +45,9 @@ def f
       ).returns(T.nilable(Array))
     }
     def run(command = nil, macos: nil, linux: nil)
+      # Save parameters for serialization
+      @run_params ||= command || { macos: macos, linux: linux }.compact
+
```

Comment:
Open to other ideas of how to handle this. Couldn't think of anything better.

Reply:
Seems fine to me!